src/java.base/unix/native/libnet/net_util_md.c

```diff
@@ -715,7 +700,7 @@ NET_Wait(JNIEnv *env, jint fd, jint flags, jint timeout)
           pfd.events |= POLLOUT;
 
         errno = 0;
-        read_rv = NET_Poll(&pfd, 1, nanoTimeout / NET_NSEC_PER_MSEC);
+        read_rv = poll(&pfd, 1, nanoTimeout / NET_NSEC_PER_MSEC);
```

Comment:
I suspect this one will need to handle EINTR like the original code.

Reply:
as far as I can tell, this method retries all poll failures regardless of errno until timeout expires; NET_Poll retried EINTR, so I think we're good here. Or am I missing something?

Reply:
I think you are right, it's ignoring all errors so will continue to loop if interrupted.